RevitLookup/RevitLookup.csproj

```diff
@@ -67,7 +67,8 @@
         <ItemGroup>
             <AddinFiles Include="$(RootDir)**\*.*" />
         </ItemGroup>
-
+        <Message Text="Executing Copy Assemblys To Addin Folder" Importance="High" />
```

Comment:
`<Message Text="$(MSBuildProjectName) -> Copying files to $(AppData)\Autodesk\Revit\Addins\$(RevitVersion)" Importance="High"/>`

Reply:
changed